### PR TITLE
Update PS2/PSP

### PR DESCRIPTION
**Pull request by rufotheone-fr**

### CODE REVIEW

**Comment by rufotheone-fr** [comment]
Most of the changes are self-explanatory.
For Metal Slug 3/4/5/6 and SamSho 5/6 removes => included in Metal Slug/SamSho Anthologies

**Comment by rufotheone-fr** [comment]
Roger that!

**Comment by rufotheone-fr** [comment]
I'm currently investigating about PES/Winning Eleven series because some sources say that games released worldwide could be "Final Evolution" instead of standard editions...

**Comment by rufotheone-fr** [comment]
Just took a look at my updates and made a mistake with the PS2.json metadata u_u
Sorry T_T

**Comment by unexpectedpanda** [comment]
I might be misunderstanding, but it looks like you've accidentally committed a deleted PS2 metadata file instead of rolling back the changes.

This (depending on the path you execute it in) should do the trick to grab the original:

`git checkout origin/master -- "metadata/Sony - PlayStation 2.json"`

You can then make a commit adding back the original file.

**Comment by rufotheone-fr** [comment]
Never really used to GitHub Desktop but it has a "Revert changes in commit" options so... ^^

File: clonelists/Sony - PlayStation 2.json [hunk 1]
```diff
@@ -3095,8 +3180,6 @@
 		],
 		"World Soccer Winning Eleven 8 - International": [
 			["Pro Evolution Soccer 4", 1],
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation 2.json:3182]: Invalid comma.

File: clonelists/Sony - PlayStation 2.json [hunk 2]
```diff
@@ -2506,10 +2577,12 @@
 		],
 		"Shin Megami Tensei - Persona 3": [
 			["Shin Megami Tensei - Persona 3 FES", 0],
-			["Persona 3 FES", 0],
 			["P3 - Persona 3", 1],
 			["Persona 3", 1]
 		],
+		"Shin Megami Tensei - Persona 3 FES": [
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation 2.json:2583]: To my understanding the previous entry was correct (zero priority titles are listed under a single title in order of priority, they aren't isolated as a separate entry for matching with their clones -- although there will be legacy entries where this old way of formatting things is in effect). Are you seeing unexpected behavior here?

A heads up that this way of formatting also likely won't be supported in Retool 2.0.

File: clonelists/Sony - PlayStation 2.json [hunk 3]
```diff
@@ -71,7 +75,8 @@
 	},
 
 	"removes": {
-        "King of Fighters 2000-2001, The (Europe)": {}
+		"King of Fighters 2000-2001, The (Europe)": {
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation 2.json:78]: I'd prefer the braces to be kept together, so it's immediately apparent that it's an empty object :)

File: clonelists/Sony - PlayStation 2.json [hunk 4]
```diff
@@ -2392,14 +2458,19 @@
...
+			["Yinghua Dazhan V Episode 0 - Huangye de Shaonue Wushi", 1]
 		],
 		"Sakura Wars - So Long, My Love (Disc 2) (Japanese Voice Over)": [
 			["Sakura Taisen V - Saraba Itoshiki Hito yo", 1]
 		],
 		"Samurai Shodown V": [
+			["Samurai Shodown Anthology", 0],
 			["Samurai Spirits Zero", 1]
 		],
+		"Samurai Spirits: Tenka-ichi Kenkakuden": [
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation 2.json:2471]: Missed a colon here, this is a dash in the file name.

File: clonelists/Sony - PlayStation 2.json [hunk 5]
```diff
@@ -2523,6 +2596,9 @@
 		"Shonen Jump's One Piece - Grand Battle": [
 			["One Piece - Grand Battle! Rush", 1]
 		],
+		"Shonen Jump's One Piece: Pirates' Carnival" :[
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation 2.json:2599]: Missed a colon here, this is a dash in the file name.

File: clonelists/Sony - PlayStation 2.json [hunk 6]
```diff
@@ -1501,6 +1520,9 @@
 		"Karaoke Revolution Party": [
 			["Karaoke Stage 2", 1]
 		],
+		"Katana Action": [
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation 2.json:1523]: This game doesn't exist in Redump's database.

File: clonelists/Sony - PlayStation Portable (Redump).json [hunk 7]
```diff
@@ -669,6 +679,9 @@
 		],
 		"Yu-Gi-Oh! GX - Tag Force 2": [
 			["Yu-Gi-Oh! Duel Monsters GX - Tag Force 2", 1]
-		]
+		],
+		"Yu-Gi-Oh! 5D's - Tag Force 5": [
+			["Yu-Gi-Oh! 5D's Tag Force 5", 1]
+		],
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation Portable (Redump).json:685]: Invalid comma here.

File: clonelists/Sony - PlayStation Portable (Redump).json [hunk 8]
```diff
@@ -542,6 +551,7 @@
 		],
 		"Shin Megami Tensei - Persona 3 Portable": [
 			["P3P - Yeosin Jeonsaeng - Persona 3 Portable", 1]
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation Portable (Redump).json:553]: Missed a comma here.

File: clonelists/Sony - PlayStation Portable (Redump).json [hunk 9]
```diff
@@ -19,7 +19,19 @@
...
+			"categories": ["Applications"]
+		},
+		"Minna no Navi (Disc 2)": {
+			"match": "short",
+			"categories": ["Applications"]
+		},
+		"Talkman": {
+			"match": "short",
+			"categories": ["Educational"]
+		},
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation Portable (Redump).json:34]: Invalid comma here. JSON doesn't support trailing commas, there must be a next entry if you include one.

File: clonelists/Sony - PlayStation Portable (Redump).json [hunk 10]
```diff
@@ -542,6 +551,7 @@
 		],
 		"Shin Megami Tensei - Persona 3 Portable": [
 			["P3P - Yeosin Jeonsaeng - Persona 3 Portable", 1]
+			["Persona 3 Portable (Japan)", 1]
```
Comment (unexpectedpanda) [inline clonelists/Sony - PlayStation Portable (Redump).json:554]: You've left the country tag in here.

**Comment by unexpectedpanda** [comment]
Big update! A few tweaks and then it should be good to go :)

File: clonelists/Sony - PlayStation 2.json [hunk 11]
```diff
@@ -12,6 +12,10 @@
 			"match": "full",
 			"categories": ["Promo", "Video"]
 		},
+		"Buzz! Escuela de Talentos": {
```
[the remark this answers is not in the document]
Comment (unexpectedpanda) [reply clonelists/Sony - PlayStation 2.json]: I've updated this in a new commit.

File: clonelists/Sony - PlayStation 2.json (hunk 2 above)
Comment (unexpectedpanda) [reply clonelists/Sony - PlayStation 2.json]: I've reverted this in a new commit so it works as intended.

**Pull request merged**